src/LinkDotNet.NCronJob/Configuration/JobOptionBuilder.cs

```diff
@@ -7,26 +7,51 @@ namespace LinkDotNet.NCronJob;
 /// </summary>
 public sealed class JobOptionBuilder
 {
-    private readonly List<JobOption> jobOptions = [];
+    private readonly ConcurrentQueue<JobOption> jobOptions = [];
```

Comment:
See last comment

Reply:
Let me know if you want to keep it as List, as indicated above it's not a current use case. It adds thread-safety but I'm sure at some minor cost. 
I can revert and run some benchmarks at a later time.